mmv1/templates/terraform/examples/secure_source_manager_instance_basic.tf.erb

```diff
@@ -4,4 +4,8 @@ resource "google_secure_source_manager_instance" "<%= ctx[:primary_resource_id]
     labels = {
       "foo" = "bar"
     }
+
+    lifecycle {
```

Comment:
Should we add a comment explaining why we are adding this block?

Reply:
Added.